static/app/views/performance/table.tsx

```diff
@@ -283,22 +287,28 @@ class _Table extends Component<Props, State> {
 
     const align = fieldAlignment(column.name, column.type, tableMeta);
     const field = {field: column.name, width: column.width};
+    const aggregateAliasTableMeta: EventsMetaType['fields'] = {};
+    if (tableMeta) {
+      Object.keys(tableMeta).forEach(key => {
+        aggregateAliasTableMeta![getAggregateAlias(key)] = tableMeta[key];
```

Comment:
Possible to avoid the `!` here? Should ew type `aggregateAliasTableMeta` with letting it be `undefined`?

Reply:
ah the `!` is not needed. `aggregateAliasTableMeta` can just remain an empty object. Removed the `!`, thanks!